api/global/internal/meter.go

```diff
@@ -50,21 +50,22 @@ type meterProvider struct {
 	delegate metric.Provider
 
 	lock   sync.Mutex
-	meters map[string]*meter
+	meters map[string]*meterEntry
```

Comment:
This should be moved to the `registry` package to unify the location of this type of functionality.

Likely for a separate PR though.

Reply:
https://github.com/open-telemetry/opentelemetry-go/issues/714

Reply:
:+1: